src/test/java/org/dependencytrack/tasks/BomUploadProcessingTaskTest.java

```diff
@@ -136,7 +130,7 @@ public void informTest() throws Exception {
         final byte[] bomBytes = Files.readAllBytes(Paths.get(getClass().getClassLoader().getResource("bom-1.xml").toURI()));
 
         new BomUploadProcessingTask().inform(new BomUploadEvent(project.getUuid(), bomBytes));
-        assertConditionWithTimeout(() -> NOTIFICATIONS.size() >= 5, Duration.ofSeconds(5));
+        assertConditionWithTimeout(() -> NOTIFICATIONS.size() >= 6, Duration.ofSeconds(5));
```

Comment:
i see this is being changed from notification size `>=  5` ... to `6`.  Is this a 1 to 1 mapping with duration of seconds?   @nscuro

Reply:
It's an inconsistency in assertions that I noticed while debugging another issue with this test. 

This statement is waiting for 6 (previously 5) notifications to be sent, and will wait up to 5 seconds for this to happen.
It's more or less a means of ensuring all asynchronous processing has completed before doing further assertions.

The actual notification content is being asserted later in the test:

https://github.com/DependencyTrack/dependency-track/blob/06168ebaa38a5b027c087e5d2d41b98aa61882a2/src/test/java/org/dependencytrack/tasks/BomUploadProcessingTaskTest.java#L155-L170

I noticed that the above assertion is checking for **6** notifications, not just 5... :D